enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/analytics/ChoreoFaultAnalyticsProvider.java

```diff
@@ -197,13 +211,11 @@ public Error getError(FaultCategory faultCategory) {
     @Override
     public String getUserAgentHeader() {
         // User agent is not required for fault scenario
```

Comment:
Provide a descriptive comment saying that this method is not called when processing fault event.